src/main/java/hudson/tasks/junit/CaseResult.java

```diff
@@ -51,11 +51,14 @@
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.logging.Logger;
+import java.util.logging.Level;import java.util.logging.Logger;
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
 
+import javax.xml.stream.*;
+import javax.xml.stream.events.*;
```

Comment:
import what you use and not star import, (you can configure the IDE to only do star import after say 99 imports from that package)